redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/EventListener.kt

```diff
@@ -124,9 +124,9 @@ public open class EventListener {
   /**
    * Invoked on a request to create an unknown layout modifier [tag].
```

Comment:
```suggestion
   * Invoked on a request to create an unknown modifier [tag].
```
Probably worth searching for "layout modifier" string, too.